harness/python/feature.py

```diff
@@ -75,12 +78,13 @@ async def run(self) -> None:
         self.client = await Client.connect(self.address, namespace=self.namespace)
 
         # Run worker
-        self.worker = Worker(
-            self.client,
-            task_queue=self.task_queue,
-            workflows=self.feature.workflows,
-            activities=self.feature.activities,
-        )
+        self.worker = self.create_worker()
+        if self.feature.worker_starter:
+            await self.feature.worker_starter(self)
+        else:
+            await self.start_worker_and_check_wf()
+
+    async def start_worker_and_check_wf(self):
```

Comment:
```suggestion
    async def start_worker_and_check_workflow(self):
```

Would prefer to avoid these Core-like abbreviations (we tend to be inconsistent in our use of them)